catalog/gitops/setters.yaml

```diff
@@ -3,12 +3,12 @@ kind: ConfigMap
 metadata:
   name: setters-config
 data:
+  # You can leave these defaults
```

Comment:
@morgante These fields will be reordered. We are working on fixing it but till then you should adjust the comments.

Reply:
@phanimarupaka I think this should be okay as its in the v1 branch. Is there an issue to track?

Reply:
Yes. https://github.com/GoogleContainerTools/kpt/issues/2332